### ISSUE TITLE
Unable to install on Linux with wxPython 4.2.0

### ISSUE THREAD

**Issue by return215** [telperion/nine-or-null#10]
First of all, I understand if Linux is not officially supported and that is fine. I am here just to share my experience.

I encountered an issue trying to compile from source on Fedora 37 with Python 3.11. When running `poetry install`, I ran into this issue (wxWidgets/Phoenix#2225) where `attrdict` module was not imported properly (for non-Windows installs only) with wxPython version 4.2.0 for the reason I can only shake my head with.

After changing the `pyproject.toml` to use the correct version (4.2.1), the issue went away, as mentioned in https://github.com/wxWidgets/Phoenix/issues/2225#issuecomment-1596190592.

```toml
[tool.poetry.dependencies]
wxPython = "^4.2.1"
```

UPDATE:
The only offending dependency remaining in my system was the GTK+ dev. After installing `gtk3-devel` for wxPython to compile, the program runs well and it can analyze the simfiles in Linux.

TL;DR: please update wxPython to 4.2.1
